docs/appendices/release-notes/unreleased.rst

```diff
@@ -63,45 +87,38 @@ Changes
   compatible type name notation with ``[]`` suffixes for arrays, instead of
   ``_array``.
 
-- Added arithmetic operation support for the :ref:`numeric <numeric_type>`.
+- Added the ``delimiter`` option for :ref:`copy_from` CSV files. The option is
+  used to specify the character that separates columns within a row.
 
-- Added support for the :ref:`numeric <numeric_type>` data type and allow the
-  ``sum`` aggregation on the :ref:`numeric <numeric_type>` type.
-  Note that the storage of the ``numeric`` data type is not supported.
+- Added the ``empty_string_as_null`` option for :ref:`copy_from` CSV files.
+  If the option is enabled, all column's values represented by an empty string,
+  including a quoted empty string, are set to ``NULL``.
 
-- Added information about the shards located on the node to the
-  :ref:`NodeInfo MXBean <node_info_mxbean>` which is available as an
-  enterprise feature.
 
-- Added the ``to_char`` scalar function for ``timestamp`` and ``interval``
-  argument data types.
+Administration and Operations
+-----------------------------
 
-- Added support for the ``split_part`` scalar function
+- Added information about the shards located on the node to the :ref:`NodeInfo
+  MXBean <node_info_mxbean>` which is available as an enterprise feature.
```

Comment:
Are :ref:`` links able to span a line break?

Reply:
Yep - see the rendered version of the PR:  https://crate--10927.org.readthedocs.build/en/10927/appendices/release-notes/unreleased.html#administration-and-operations